target_chains/ethereum/contracts/contracts/pulse/IPulse.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "./PulseEvents.sol";
+import "./PulseState.sol";
+
+interface IPulseConsumer {
+    function pulseCallback(
+        uint64 sequenceNumber,
+        address updater,
+        PythStructs.PriceFeed[] memory priceFeeds
+    ) external;
+}
+
+interface IPulse is PulseEvents {
+    // Core functions
+    /**
+     * @notice Requests price updates with a callback
+     * @dev The msg.value must cover both the Pyth fee and gas costs
+     * Note: The actual gas required for execution will be 1.5x the callbackGasLimit
```

Comment:
This comment will be confusing to users as they don't know how to determine the fee. I think the comment should say msg.value should be set to `getFee(callbackGasLimit)`